docs/src/rules/no-restricted-imports.md

```diff
@@ -237,137 +202,147 @@ import * as Foo from "foo";
 
 :::
 
-::: incorrect { "sourceType": "module" }
+Examples of **correct** code for `importNames` in `paths`:
+
+Other strings doesn't report default exported import even if the names are same:
```

Comment:
I'm not entirely sure what this means. Maybe this?
```suggestion
If the local name assigned to a default export is the same as a string in `importNames`, this will not cause an error.
```

Reply:
Exactly this.